tests/filtered_relation/models.py

```diff
@@ -0,0 +1,110 @@
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Author(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    favourite_books = models.ManyToManyField(
+        'Book',
+        related_name='preferred_by_authors',
+        related_query_name='preferred_by_authors')
```

Comment:
This line needs a trailing comma and the ) goes on the next line. Please use the same hanging indent style throughout this file.